cumulusci.yml

```diff
@@ -133,6 +163,17 @@ tasks:
             path: unpackaged/config/customer_profiles
 
 flows:
+    make_community:
```

Comment:
@bethbrains , One question here. Do you envision calling this flow separately only when we have to test experience cloud.

Reply:
Yes

Reply:
👍